sdk/core/azure-core/src/main/java/com/azure/core/util/polling/LocationPollingStrategy.java

```diff
@@ -59,8 +61,21 @@ public LocationPollingStrategy(HttpPipeline httpPipeline) {
      * @throws NullPointerException If {@code httpPipeline} is null.
      */
     public LocationPollingStrategy(HttpPipeline httpPipeline, ObjectSerializer serializer) {
+        this(Objects.requireNonNull(httpPipeline, "'httpPipeline' cannot be null"), (serializer == null) ? DEFAULT_SERIALIZER : serializer, Context.NONE);
```

Comment:
You can just call the other overload without doing any validation here because the other overload is doing the same validation.
```suggestion
        this(httpPipeline, serializer, Context.NONE);
```